tests/mmptest/src/NativeReferencesTests.cs

```diff
@@ -10,10 +10,10 @@
 namespace Xamarin.MMP.Tests
 {
 	public partial class MMPTests {
```

Comment:
These changes are needed for BindingProjectTests to be outside of the meta-MMPTests, which is a test organization change i'm making in bite sized pieces.